src/main/java/org/snomed/aag/data/services/ProjectAcceptanceCriteriaService.java

```diff
@@ -41,7 +43,7 @@
     private BranchSecurityService branchSecurityService;
 
 	private static void verifyParams(String branchPath, Integer projectIteration) {
-        if (branchPath == null || projectIteration == null || projectIteration < 0) {
+        if (branchPath == null || (projectIteration != null && projectIteration < 0)) {
```

Comment:
projectIteration can now be null (for tasks) so various conditions had to be changed.